build-tools/packages/build-tools/src/repoPolicyCheck/handlers/fluidBuildTasks.ts

```diff
@@ -448,7 +451,7 @@ export const handlers: Handler[] = [
 			for (const script in json.scripts) {
 				// eslint-disable-next-line @typescript-eslint/no-non-null-assertion
 				const command = json.scripts[script]!;
-				if (command.startsWith("tsc") && !ignore.has(script)) {
+				if ((command.startsWith("tsc ") || command === "tsc") && !ignore.has(script)) {
```

Comment:
Can we add a brief comment explaining the updated check here? Presumably this is to avoid matching on script names that start with "tsc" (e.g. `tsc-multi`)